src/Take.Elephant.sln

```diff
@@ -29,10 +29,12 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Take.Elephant.Azure", "Take
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Take.Elephant.Kafka", "Take.Elephant.Kafka\Take.Elephant.Kafka.csproj", "{F5132A9D-6674-40E3-AB1D-6252C49BD23D}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Take.Elephant.Elasticsearch", "Take.Elephant.ElasticSearch\Take.Elephant.ElasticSearch.csproj", "{4CAFB6C5-9F17-46FA-90A8-621DA1C91423}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Take.Elephant.ElasticSearch", "Take.Elephant.ElasticSearch\Take.Elephant.ElasticSearch.csproj", "{4CAFB6C5-9F17-46FA-90A8-621DA1C91423}"
```

Comment:
I s this a breaking change? will it change our package naming?

Reply:
True